### DISCUSSION TITLE
Not sure if this is a bug, but this behavior related to tab omnibar is unexpected.  Hoping someone can help me.  Thanks!

### DISCUSSION THREAD

**Discussion by chriswu000** [brookhong/Surfingkeys#1481] [Q&A]
When I type `T`, I expect the tab search omnibar.  Instead, every once in a while, I'll start getting this set of links at the top of Chrome:

<img width="1790" alt="Screen Shot 2021-06-14 at 3 39 34 PM" src="https://user-images.githubusercontent.com/889996/121949824-ee931500-cd26-11eb-8809-f89c5dc1077e.png">

I can't get it to consistently stop doing this.  Any help would be great, thanks!

**Comment by Shamaoke** [comment]
That is another possible way for choosing tabs. By default, there exists a threshold. When you cross it, the Omnibar is used.

The following is how you can completely disable the feature.
   
    // set a threshold for choosing tabs with Omnibar (0 for always use it)
    settings.tabsThreshold = 0
